src/ontology/cl-edit.owl

```diff
@@ -19054,13 +19054,20 @@ SubClassOf(obo:CL_0002451 obo:CL_0000048)
 AnnotationAssertion(oboInOwl:created_by obo:CL_0002453 "tmeehan"^^xsd:string)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002453 "2010-11-10T02:51:34Z"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24133281"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "O-2A/OPC"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11756508"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "O2A/OPC"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:2253328"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "oligodendrocyte-type 2 astrocyte (O-2A) progenitor cell"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11756508"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "oligodendrocyte-type-2 astrocyte progenitor cell"^^xsd:string)
+AnnotationAssertion(Annotation(<http://purl.org/dc/terms/contributor> "http://orcid.org/0000-0002-0819-0473") Annotation(oboInOwl:hasDbXref "PMID:24133281"^^xsd:string) oboInOwl:hasBroadSynonym obo:CL_0002453 "oligodendrocyte/type-2 astrocyte progenitor cell"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0002453 "cell"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10704434"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:2182078"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://veteriankey.com/of-the-central-and-peripheral-nervous-system/"^^xsd:string) rdfs:comment obo:CL_0002453 "This cell type can be purified from optic nerves and other regions of the embryonic, postnatal and adult rat CNS and can be differentiated in vitro to oligodendrocytes and type-2 astrocytes. Some references use the terms 'oligodendrocyte precursor cell' and 'oligodendrocyte type-2 astrocyte (O-2A) progenitor' to refer to the same entity (e.g., PMID:10704434), while others describe an oligodendrocyte type-2 astrocyte (O-2A) progenitor cell as the precurosr to oligodendrocyte- and type-2 astrocyte- progenitor cells (e.g, https://veteriankey.com/of-the-central-and-peripheral-nervous-system/)."^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002453 "oligodendrocyte precursor cell"^^xsd:string)
-EquivalentClasses(obo:CL_0002453 ObjectIntersectionOf(obo:CL_0000123 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001017)))
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0002453 obo:CL_0000123)
+SubClassOf(obo:CL_0002453 ObjectIntersectionOf(obo:CL_0000123 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001017)))
+SubClassOf(obo:CL_0002453 ObjectIntersectionOf(obo:CL_0000123 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127)))
```

Comment:
I don't think you should have two develops into - this would mean that OPCs develop into both astrocytes and oligodendrocytes. (all statements must be true) - the opposite develops from might be a better way to axiomatize this (eg type-2 astrocytes develop from OPC and oligodendrocytes develop from OPCs)

Reply:
Removed SubClass Of axioms. 
Oligodendrocyte already has recommended axiom.
Type-2 astrocyte does not exist in CL. Create this new term now vs submit separate NTR ticket?

Reply:
100% new NTR ticket - if not discussions about that will stop this ticket from going through lol